### PR TITLE
PIM-5800: Proofread and update the 1.6 technical doc

### PR DESCRIPTION
**Pull request by willy-ahva**
Several fixes about links.

Update doc about Notification that has been deprecated.

### CODE REVIEW

File: cookbook/notification/how_to_add_a_notification.rst [hunk 1]
```diff
@@ -83,13 +83,77 @@ The optional route parameter will be:
...
-        ]
-    ];
+    $route = 'pim_importexport_export_execution_show';
+    $routeParams = ['id' => $jobExecutionId];
 
 If the messageParams option is provided, it will be passed to the message when translating it.
 
 The context allows to store some extra data in the notification, it is not displayed in the UI by default.
+
+Now register your new factory with the good tag:
```
Comment (solivier) [inline cookbook/notification/how_to_add_a_notification.rst]: proper tag

File: cookbook/notification/how_to_add_a_notification.rst [hunk 2]
```diff
@@ -3,70 +3,70 @@ How to Add a Notification
...
+        {
+            $this->notificationTypes = $notificationTypes;
+        }
+
+        /**
+         * {@inheritdoc}
+         */
+        public function create($status, $message, $messageParams, $route, $routeParams, $type)
+        {
+            // Here is a simple implementation of create method. You can add your custom code.
```
Comment (rybus) [inline cookbook/notification/how_to_add_a_notification.rst]: of the `create()` method, you can add your custom code.

File: cookbook/notification/how_to_add_a_notification.rst [hunk 3]
```diff
@@ -83,13 +83,77 @@ The optional route parameter will be:
...
+    services:
+        acme_custom.event_subscriber.custom_subscriber:
+            class: 'Acme\CustomBundle\EventSubscriber\CustomEventSubscriber'
+            arguments:
+                - '@pim_notification.registry.factory.notification'
+                - '@pim_notification.notifier'
+            tags:
+                - { name: kernel.event_subscriber }
+
+Here, we inject services in an event subscriber, but you can inject them wherever you have an action to notify a user.
```
Comment (rybus) [inline cookbook/notification/how_to_add_a_notification.rst]: which notifies a user

**Comment by rybus** [comment]
Tiny things to fix then GTM

File: cookbook/notification/how_to_add_a_notification.rst [hunk 4]
```diff
@@ -3,70 +3,70 @@ How to Add a Notification
...
 The Akeneo PIM comes with a notification system.
 
-In a simple way, you can add your notifications to the notifier widget.
-
 Prerequisites
 -------------
 
-The Akeneo PIM project introduces services to help you manage your notifications.
+If you need to send a custom message for an existing notification type (like import/export, mass edit, etc.)
+you can go to the TODO step.
```
Comment (nidup) [inline cookbook/notification/how_to_add_a_notification.rst]: TODO step?

**Comment by MarieMinasyan** [comment]
GTM

**Comment by solivier** [comment]
gtm

**Pull request merged**